draft-reddy-tls-slhdsa.md

```diff
@@ -154,25 +164,11 @@ SLH-DSA imposes an upper bound of 2^64 signatures per key. If a key pair were us
 
 ## Key Lifetime Management
 
-Certificates are typically issued with finite lifetimes, and upon rotation, a new key-pair must be generated to obtain a new certificate. For example, if a server certificate is valid for 1 year and each client connection re-authenticates every 12 hours, only 730 signatures would be generated per client. 
-This implies that a single SLH-DSA key could theoretically support up to 2^64 / 730 ≈ 2.52 × 10^16 clients over a certificate's lifetime.
-
 In order to maintain cryptographic safety in high-scale environments, deployments MUST:
 
    *  Rotate SLH-DSA certificates and keys periodically;
```

Comment:
What does "periodically" mean?  Once a day?  Once a century?

Reply:
Good point, I propose the replace the text as follows:

Rotate SLH-DSA certificates and keys based on expected signature usage, ensuring ample margin from the 2⁶⁴ signature limit.